frontend/src/views/ManagePermissionsView/index.vue

```diff
@@ -0,0 +1,329 @@
+<script setup lang="ts">
+import { computed, onUnmounted, ref, watch, type Component } from "vue";
+import { useRoute, useRouter } from "vue-router";
+import { isAxiosError } from "axios";
+import { useQuery, useQueryClient } from "@tanstack/vue-query";
+import type { DropdownChangeEvent } from "primevue/dropdown";
+import TabView, { type TabViewChangeEvent } from "primevue/tabview";
+import TabPanel from "primevue/tabpanel";
+import EnterpriseIcon from "@carbon/icons-vue/es/enterprise/16";
+import UserIcon from "@carbon/icons-vue/es/user/16";
+import { AdminRoleAuthGroup } from "fam-admin-mgmt-api/model";
+import PageTitle from "@/components/UI/PageTitle.vue";
+import { AdminMgmtApiService } from "@/services/ApiServiceFactory";
+import { EnvironmentSettings } from "@/services/EnvironmentSettings";
+import Dropdown from "@/components/UI/Dropdown.vue";
+import TablePlaceholder from "@/components/ManagePermissionsTable/TablePlaceholder.vue";
+import {
+    selectedApp,
+    setSelectedApp,
+    activeIndex,
+} from "@/store/ApplicationState";
+import {
+    formatAxiosError,
+    getUniqueApplications,
+    isSelectedAppAuthorized,
+} from "@/utils/ApiUtils";
+import ManagePermissionsTable from "@/components/ManagePermissionsTable";
+import type {
+    PermissionNotificationType,
+    ManagePermissionsTabHeaderType,
+    ManagePermissionsTabTypes,
+} from "@/types/ManagePermissionsTypes";
+import NotificationStack from "@/views/ManagePermissionsView/NotificationStack.vue";
+import {
+    AppAdminErrorQuerykey,
+    AppAdminSuccessQuerykey,
+    DelegatedAdminErrorQueryKey,
+    DelegatedAdminSuccessQueryKey,
+    type AppPermissionQueryErrorType,
+} from "../AddAppPermission/utils";
+import type { FamUserRoleAssignmentRes } from "fam-app-acsctl-api/model";
+import {
+    generateAppErrorNotifications,
+    generateAppSuccessNotifications,
+    generateFamNotification,
+} from "./utils";
+import {
+    FamPermissionErrorQueryKey,
+    FamPermissionSuccessQueryKey,
+} from "../AddFamPermission/utils";
+
+const queryClient = useQueryClient();
+const route = useRoute();
+const router = useRouter();
+const appIdFromQueryParam = ref(route.query.appId);
+
+const environment = new EnvironmentSettings();
+
+// Fetch notification Data
+const appAdminSuccessData = queryClient.getQueryData<FamUserRoleAssignmentRes>([
+    AppAdminSuccessQuerykey,
+]);
+const appAdminErrorData = queryClient.getQueryData<AppPermissionQueryErrorType>(
+    [AppAdminErrorQuerykey]
+);
+const delegatedAdminSuccessData =
+    queryClient.getQueryData<FamUserRoleAssignmentRes>([
+        DelegatedAdminSuccessQueryKey,
+    ]);
+const delegatedAdminErrorData =
+    queryClient.getQueryData<AppPermissionQueryErrorType>([
+        DelegatedAdminErrorQueryKey,
+    ]);
+const FamPermissionSuccessData = queryClient.getQueryData<string>([
+    FamPermissionSuccessQueryKey,
+]);
+const FamPermissionErrorData = queryClient.getQueryData<string>([
+    FamPermissionErrorQueryKey,
+]);
```

Comment:
renamed variables instead of adding more comments, the variable names should make more sense now https://github.com/bcgov/nr-forests-access-management/pull/1650/commits/4dc6bac3e46791aed75f8349a8a8517a7a77a8e2